docs/csharp/how-to/how-to-concatenate-multiple-strings.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "How to: Concatenate Multiple Strings (C# Guide)"
+description: There are multiple ways to concatenate strings in C#. Learn the options and the reasons behind different choices.
+ms.date: 01/11/2018
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "joining strings [C#]"
+  - "concatenating strings [C#]"
+  - "strings [C#], concatenation"
+ms.assetid: 8e16736f-4096-4f3f-be0f-9d4c3ff63520
+caps.latest.revision: 21
+author: "BillWagner"
+ms.author: "wiwagn"
+---
+# How to: Concatenate Multiple Strings (C# Guide)
+
+*Concatenation* is the process of appending one string to the end of another string. You concatenate strings by using the + operator. For string literals and string constants, concatenation occurs at compile time; no run-time concatenation occurs. For string variables, concatenation occurs only at run time.
+
+The following example uses concatenation to split a long string literal into smaller strings in order to improve readability in the source code. These parts will be concatenated into a single string at compile time. There is no run-time performance cost regardless of the number of strings involved.  
+  
+ [!code-csharp-interactive[Combining strings at compile time](../../../samples/snippets/csharp/how-to/strings/Concatenate.cs#1)]  
+  
+
+To concatenate string variables, you can use the `+` or `+=` operators, or the <xref:System.String.Concat%2A?displayProperty=nameWithType>, <xref:System.String.Format%2A?displayProperty=nameWithType> or <xref:System.Text.StringBuilder.Append%2A?displayProperty=nameWithType> methods. The `+` operator is easy to use and makes for intuitive code. Even if you use several + operators in one statement, the string content is copied only once. The following code shows two examples of using the `+` operator to concatenate
```

Comment:
Since you mention it in line 32, string interpolation should also appear in this list.